src/object/srv_obj.c

```diff
@@ -1023,11 +1000,11 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 	 * modification or not, so still need to dispatch
 	 * the RPC to other replicas.
 	 */
-	rc = dtx_begin(&orw->orw_dti, &orw->orw_oid, cont->sc_hdl,
-		       orw->orw_epoch, orw->orw_dkey_hash,
-		       &conflict, orw->orw_dti_cos.ca_arrays,
-		       orw->orw_dti_cos.ca_count, orw->orw_map_ver,
-		       DAOS_INTENT_UPDATE, true, &dth);
+	rc = dtx_leader_begin(&orw->orw_dti, &orw->orw_oid, cont->sc_hdl,
+			      orw->orw_epoch, orw->orw_dkey_hash,
+			      orw->orw_map_ver, DAOS_INTENT_UPDATE,
+		  	      orw->orw_shard_tgts.ca_arrays,
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs